index.js

```diff
@@ -181,3 +185,55 @@ AddonManager.addInstallListener({
     app.send('addon-install:download-failed', formatInstallData(install));
   }
 });
+
+function pingServer(title, addon) {
+  cookieManager.getCookiesFromHost('ideatown.dev', function(cookies) {
+    let csrf;
+    let cookieStr = '';
+    cookies.forEach(function(c) {
+      cookieStr += c.name + '=' + c.value + ' ';
+      if (c.name === 'csrftoken') csrf = c.value;
+    });
+
+    if (csrf) {
+      getMe(csrf, cookieStr, function(id) {
+        request({
+          url: prefs.METRICS_URL,
+          content: formatEvent(title, id, addon),
+          headers: {
+            'Cookie': cookieStr,
+            'X-CSRFToken': csrf
```

Comment:
If we are passing all the cookies in the `Cookie` header, that should include the `csrftoken` cookie; do we still need to separately set `X-CSRFToken`?